etc/reference.config

```diff
@@ -1,26 +1,42 @@
 # config reference: system URL
 
-config reference: bugtraq   http://www.securityfocus.com/bid/
-config reference: bid	    http://www.securityfocus.com/bid/
-config reference: cve       http://cve.mitre.org/cgi-bin/cvename.cgi?name=
-#config reference: cve       http://cvedetails.com/cve/
-config reference: secunia   http://www.secunia.com/advisories/
+##############################
+# Referenced by ET/Open ET/Pro
+##############################
+
+#  resolves, works as intended
+config reference: cve       https://cve.mitre.org/cgi-bin/cvename.cgi?name=
+config reference: nessus    https://www.tenable.com/plugins/nessus/
+config reference: url       https://
```

Comment:
I see some rules / rulesets that use a full url here, so including the https://. These would end up having the https:// twice. Perhaps it would be good to be forgiving about this.

Reply:
yeah but w/o logging it wasn't as issue, now it will be

I think it would be good to correct for it, ya